README.md

```diff
@@ -21,7 +21,7 @@ $ composer require php-http/discovery
 
 ## Documentation
 
-Please see the [official documentation](http://php-http.readthedocs.org/en/latest/discovery/).
+Please see the [official documentation](http://php-http.readthedocs.org/en/latest/discovery.html).
```

Comment:
sorry, i was too fast. i guess we also need to fix this over the other
repositories.